### DISCUSSION TITLE
Custom attribute for `grid-cols-*` not generating valid CSS

### DISCUSSION THREAD

**Discussion by charliegroll** [tailwindlabs/tailwindcss#7936] [Help] [answered]
I'm trying to use the class `lg:tw-grid-cols-[64fr_36fr]` but that doesn't seem to generate valid CSS (the underscore isn't replaced). 
<img width="288" alt="image" src="https://user-images.githubusercontent.com/1126370/159965497-7db442c6-dccb-4559-9e88-47cbc1c09315.png">

Tailwind `2.2.19`
`mode: 'jit'`
`prefix: 'tw-'`

is this a known bug that won't be addressed in v2?

**Comment by adamwathan** [comment, marked as answer]
Yeah you should use v3 if you want to use the JIT features, that's where all development of that stuff happens now 👍🏻 

You might be able to use `tw-grid-col[64fr,36fr]` — we used commas in v2 for a while before adding the underscore stuff.

**Reply by charliegroll** [reply]
`tw-grid-col[64fr,36fr]` works like a charm, thanks!
